com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -2603,14 +2603,6 @@ private void HandleSessionOwnerEvent(uint sceneEventId, ulong clientId)
                         // Mark this client as being connected
                         NetworkManager.ConnectedClients[clientId].IsConnected = true;
 
-                        // Notify the local server that a client has finished synchronizing
```

Comment:
I think this needs to stay here?
Only clients in a distributed authority topology will invoke the HandleSessionOwnerEvent, but they also need to be notified of all scene events in order to stay synchronized with the current session.

Let's walk through this so I can better understand your thoughts/direction on this?